docs/contribution.md

```diff
@@ -39,3 +39,86 @@ pip install 'anta[dev]'
 ```
 
 > This command has to be done after you install repository with commands provided in previous section.
+
+Then, tox is configued with few environment to run CI locally:
+
+```bash
+tox list
+default environments:
+clean  -> run the test driver with /home/tom/.pyenv/versions/3.9.9/envs/arista-anta/bin/python3.9
+py38   -> run the test driver with py38
+py39   -> run the test driver with py39
+py310  -> run the test driver with py310
+lint   -> check the code style
+type   -> check typing
+report -> run the test driver with /home/tom/.pyenv/versions/3.9.9/envs/arista-anta/bin/python3.9
+
+additional environments:
+3.8    -> run the test driver with 3.8
+3.9    -> run the test driver with 3.9
+3.10   -> run the test driver with 3.10
```

Comment:
these additional env are for github but this is the output of the command so yeah.. :p